### ISSUE TITLE
Resolve table header accessibility violations

### ISSUE THREAD

**Issue by lawhorkl** [uclibs/scholar_uc#1591]
### Descriptive summary

Our recent accessibility scan of Scholar 3 reveals 105 total accessibility issues, 6 of these are issues with our data table headers not referring to data table cells.

We need to resolve them before we push to prod.

A good tool while working on this is https://achecker.ca/checker/index.php to allow you to make sure your changes solve the issue.

The following is a list of all data table accessibility issues:

* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044402
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044410
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044418
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044588
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3044955
* https://wsuofc.dequecloud.com/worldspace/issue/edit/3045562
